statsmodels/genmod/families/family.py

```diff
@@ -505,19 +543,29 @@ def loglike(self, endog, mu, freq_weights=1., scale=1.):
         -------
         llf : float
             The value of the loglikelihood function evaluated at
-            (endog,mu,scale) as defined below.
+            (endog,mu,freq_weights,scale) as defined below.
 
         Notes
         -----
         If the link is the identity link function then the
         loglikelihood function is the same as the classical OLS model.
-        llf = -(nobs/2)*(log(SSR) + (1 + log(2*pi/nobs)))
-        where SSR = sum((endog-link^(-1)(mu))**2)
+
+        .. math::
+
+           llf = -nobs / 2 * (\log(SSR) + (1 + \log(2 \pi / nobs)))
+
+        where
+
+        .. math::
+           SSR = \sum_i (Y_i - g^{-1}(\mu_i))^2
 
         If the links is not the identity link then the loglikelihood
         function is defined as
-        llf = sum((`endog`*`mu`-`mu`**2/2)/`scale` - `endog`**2/(2*`scale`) - \
-            (1/2.)*log(2*pi*`scale`))
+
+        .. math::
+
+           llf = \sum_i freq\_weights_i * ((Y_i * \mu_i - \mu_i^2 / 2) / \phi -
+           Y^2 / 2 * \phi - (1/2) * \log(2 * \pi * \phi))
```

Comment:
indent either to sum_i or to freq.
Should the be braces `{}` or parenthesis around the entire expression in the sum
I always find it confusing in papers if authors don't do that.